### ISSUE TITLE
manual pylama path selection doesnt work

### ISSUE THREAD

**Issue by tom-spalding** [pchomik/linter-python#56]
the manual path selection doesnt appear to work, and I'm not seeing any sort of debug infos on why. I turned on the "force" option, "debug" option, and closing-and-reopening atom. I also tried with the "pylint" option turned on and off to see if anything different happened.

![image](https://user-images.githubusercontent.com/2218331/51792612-594c5180-2168-11e9-947e-0d70b46960c1.png)

could this because `apm` is reporting `2.7.15+`, but `pylint` was installed with `pip3`? that is; `pip3 install pylama pylama-pylint pylint` along with `apm install linter-python`.

```
> pylint --version
pylint 2.2.2
astroid 2.1.0
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0]
```

```
> apm --version
apm  2.1.3
npm  6.2.0
node 8.9.3 x64
atom 1.35.0-beta0
python 2.7.15+
git 2.19.1
```

also, is automatic `pylama` detection possible? I'm using the location provided by `which`:

```
> which pylama
/home/alligator/.local/bin/pylama
```

thanks!

edit: also note that one of the most common ways for using python3 as default is to add an alias, `alias python=python3`, however atom doesnt utilize the `python` command so even though I am using that alias, `apm --version` will still report the above. just noting for anyone else who has the same setup.

```
> python --version
Python 3.6.7
```
